nomad/structs/structs.go

```diff
@@ -1482,6 +1483,18 @@ type NodeConnQueryResponse struct {
 	QueryMeta
 }
 
+// HostDataRequest
```

Comment:
More comments please!

Reply:
Whoops, just didn't finish those fixed in facdd506c